book.rb

```diff
@@ -1,17 +1,14 @@
-require_relative 'rental'
-
 class Book
-  attr_accessor :title, :author
-  attr_reader :rentals
+  attr_accessor :title, :author, :rentals
 
   def initialize(title, author)
     @title = title
     @author = author
     @rentals = []
   end
 
-  def add_rental(date, person)
-    rental = Rental.new(date, person, self)
-    @rentals << rental unless rentals.includes?(rental)
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.book = self
   end
 end
```

Comment:
- [ ] Also after creating the book, when the user lists all books the added book does not show and the app terminates, kindly fix this as it has been introduced with the new commits

Screenshot
![image](https://github.com/Ademola-coding/School-Library/assets/81932478/55f6dd0c-594b-4567-8779-62c06e832c8a)


- The program should not exit after creating a book and a person, kindly fix this as it has been introduced with the new commits
Before new commits
![image](https://github.com/Ademola-coding/School-Library/assets/81932478/15ca6cf5-77c9-45a0-9a84-f10ffe5debd7)

Reply:
Bugs identified and are all fixed

Reply:
**Adding Book and person**

![Screenshot (96)](https://github.com/Ademola-coding/School-Library/assets/96092850/ffbdf4b6-7ecb-4535-9e73-4b2d0f8da639)

![Screenshot (97)](https://github.com/Ademola-coding/School-Library/assets/96092850/a4b344ed-b090-4338-a91f-d76906045c48)

**Book Added in JSON file**

![Screenshot (99)](https://github.com/Ademola-coding/School-Library/assets/96092850/906f7a56-2196-4076-b59f-95a9861c266d)